db/migrate/20220812082116_create_gossip_nodes.rb

```diff
@@ -0,0 +1,15 @@
+class CreateGossipNodes < ActiveRecord::Migration[6.1]
+  def change
+    create_table :gossip_nodes do |t|
+      t.string :identity
+      t.string :network
+      t.string :ip
+      t.integer :tpu_port
+      t.integer :gossip_port
+      t.string :version
```

Comment:
Is it software_version? Then call it software_version as it it called in validators. There's a "version" column in ValidatorIp table, but it's something completely different, so it's confusing to use this name here. 
(if it doesn't refer to software version then please ignore this comment)

Reply:
@maciejkocylapc